extensions/dicom-p10-downloader/src/commandsModule.js

```diff
@@ -0,0 +1,85 @@
+import OHIF from '@ohif/core';
+import {
+  save,
+  getDicomWebClientFromContext,
+  getSOPInstanceReferenceFromActiveViewport,
+  getSOPInstanceReferencesFromViewports,
+} from './utils';
+import downloadAndZip from './downloadAndZip';
+
+export function getCommands(context) {
+  const actions = {
+    // Example for running this command using Commands Manager
+    // commandsManager.runCommand(
+    //   'downloadAndZip',
+    //   {
+    //     listOfUIDs: [...],
+    //     options: {
+    //       progress(status) {
+    //         console.info('Progress:', (status.progress * 100).toFixed(2) + '%');
+    //       }
+    //     }
+    //   },
+    //   'VIEWER'
+    // );
+    downloadAndZip({ servers, dicomWebClient, listOfUIDs, options }) {
+      return save(
+        downloadAndZip(
```

Comment:
Can you import this as an alias? Otherwise this function is hard to follow and looks like it is recursive.

Reply:
I see... Yes, I can do that.